app/controllers/v0/institutions_controller.rb

```diff
@@ -68,6 +68,12 @@ def search_results
                  .filter(:eight_keys, @query[:eight_keys_to_veteran_success]) # boolean
     end
 
+    # rubocop:disable Style/MutableConstant
```

Comment:
Rubocop wants a `.freeze` on this, but then that causes an error somewhere in the JSON serialization of the response.